tools/rng_test/rng_test.py

```diff
@@ -0,0 +1,107 @@
+from operator import eq
+
+import serial
+import time
+import threading
+import sys
+
+# ------------- Global Variables -------------
+port_num = '/dev/ttyUSB1'       # port number
+baudrate = 115200               # baudrate
+
+iterator = 400                  # Number of iteration
```

Comment:
Maybe it would be nice if we can pass the three as arguments to the script instead of asking the user to edit them if needed.

Reply:
I changed the code according to your advise. Thank you for review.